JoshuaOndieki/ridemyway-api
# [Delivers #158791587] Edit User API Endpoint

**What does this PR do?**
Adds edit user endpoint to the API

**Description of Task to be completed**
A user should be able to edit their details

PUT `/api/v2/users`

**Tasks achieved in this PR**
- [x] PUT `/api/v2/users`


**How should one test this endpoint?**
- Clone this repo using `git clone https://github.com/JoshuaOndieki/ridemyway-api.git`
- Check the ReadMe for instructions on setting up the project. 
- `git checkout ft-edit-user-158791587` 
- Run `nosetests` to see the tests pass. Alternatively, run `pytest`

**How can you help?**
Review my work and let me know any semantic, technical or syntax/standards errors I may have.
I will greatly appreciate any feature/change suggestions too :smiley: 

**Resources & Libs**
[This Pivotal Tracker board](https://www.pivotaltracker.com/n/projects/2179581) was used to plan and manage the tasks.

**Relevant PIvotal Tracker Stories?**
[#158791587](https://www.pivotaltracker.com/story/show/158791587)
